lib/serializer/src/operations.js

```diff
@@ -339,63 +339,74 @@ export const ticket_update_operation_fee_parameters = new Serializer("ticket_upd
 });
 
 export const personal_data_create_operation_fee_parameters = new Serializer(
-    "personal_data_create_operation_fee_parameters",
-    {}
+    "personal_data_create_operation_fee_parameters", {
+        fee: uint64,
+        price_per_kbyte: uint32
+    }
 );
 
 export const personal_data_remove_operation_fee_parameters = new Serializer(
-    "personal_data_remove_operation_fee_parameters",
-    {}
+    "personal_data_remove_operation_fee_parameters", {
+        fee: uint64
+    }
 );
 
 export const content_card_create_operation_fee_parameters = new Serializer(
-    "content_card_create_operation_fee_parameters",
-    {}
+    "content_card_create_operation_fee_parameters", {
+        fee: uint64,
+        price_per_kbyte: uint32
+    }
 );
 
 export const content_card_update_operation_fee_parameters = new Serializer(
-    "content_card_update_operation_fee_parameters",
-    {}
+    "content_card_update_operation_fee_parameters", {
+        fee: uint64,
+        price_per_kbyte: uint32
+    }
 );
 
 export const content_card_remove_operation_fee_parameters = new Serializer(
-    "content_card_remove_operation_fee_parameters",
-    {}
+    "content_card_remove_operation_fee_parameters", {
+        fee: uint64
+    }
 );
 
 export const permission_create_operation_fee_parameters = new Serializer(
-    "permission_create_operation_fee_parameters",
-    {}
+    "permission_create_operation_fee_parameters", {
+        fee: uint64
+    }
 );
 
 export const permission_remove_operation_fee_parameters = new Serializer(
-    "permission_remove_operation_fee_parameters",
-    {}
+    "permission_remove_operation_fee_parameters", {
+        fee: uint64
+    }
 );
 
 export const content_vote_create_operation_fee_parameters = new Serializer(
-    "content_vote_create_operation_fee_parameters",
-    {}
+    "content_vote_create_operation_fee_parameters", {
+        fee: uint64
+    }
 );
 
 export const content_vote_remove_operation_fee_parameters = new Serializer(
-    "content_vote_remove_operation_fee_parameters",
-    {}
+    "content_vote_remove_operation_fee_parameters", {
+        fee: uint64
+    }
 );
 
 export const vote_counter_update_operation_fee_parameters = new Serializer(
-    "vote_counter_update_operation_fee_parameters",
-    {}
+    "vote_counter_update_operation_fee_parameters", {
+        fee: uint64
+    }
 );
 
 export const commit_create_operation_fee_parameters = new Serializer(
-    "commit_create_operation_fee_parameters",
-    {}
+    "commit_create_operation_fee_parameters"
```

Comment:
Repeat this for v2 and v3 commit/reveal operations - they are essentially the same.

Reply:
Done